pytest-embedded-idf/pytest_embedded_idf/app.py

```diff
@@ -214,7 +214,8 @@ def get_sha256(self, filepath: str) -> Optional[str]:
         Returns:
             sha256 appended to app
         """
-        from esptool import LoadFirmwareImage, hexify
+        from esptool.bin_image import LoadFirmwareImage
+        from esptool.util import hexify
```

Comment:
could you use the `self.ESPTOOL_VERSION == EsptoolVersion.V3` or `self.ESPTOOL_VERSION == EsptoolVersion.V4` to determine the correct import here? We still need to make the code compatible with esptool 3.x.

Reply:
Updated the PR